src/main/kotlin/com/bonitasoft/gradle/maven/settings/MavenSettingsPlugin.kt

```diff
@@ -128,24 +129,24 @@ class MavenSettingsPlugin : Plugin<Project> {
     }
 
     private fun registerMirrors(project: Project) {
+        val mirrorsToAdd = mutableMapOf<String, Action<MavenArtifactRepository>>()
         project.repositories.toList().filter { repo ->
             if (repo.name.equals(ArtifactRepositoryContainer.DEFAULT_MAVEN_LOCAL_REPO_NAME)) {
                 return@filter false
             }
             repo.getMirror(settings)?.also { mirror ->
```

Comment:
Am I correct if we add this comment?
```suggestion
            // If the current repository has a mirror defined in maven's settings.xml:
            repo.getMirror(settings)?.also { mirror ->
```